docs/apis-tools/tasklist-api-rest/schemas/requests/task-search-request.mdx

```diff
@@ -178,19 +178,23 @@ TaskSearchRequest - query object to search tasks by provided params.
 
 #### [<code style={{ fontWeight: 'normal' }}>TaskSearchRequest.<b>searchAfter</b></code>](#)<Bullet />`string` <Badge class="secondary" text="list"/>
 
-> Array of values copied from `sortValues` of one of the tasks, query will return page of tasks going directly after this values plus same sort values.
+> Array of values that should be copied from `sortValues` of one of the tasks from the current search results page.
```

Comment:
```suggestion
> Used to return a paginated result. Array of values that should be copied from `sortValues` of one of the tasks from the current search results page.
```

Let's add this sentence to all the 4 options to make the purpose of it clear, and allow users to find quickly how to paginate with the API (it's a common question)

Reply:
updated for the 4 fields